localization/l10n/bundle.l10n.json

```diff
@@ -238,6 +238,16 @@
   "Expand All": "Expand All",
   "Collapse All": "Collapse All",
   "Filter for any field...": "Filter for any field...",
+  "Microsoft would like your feedback": "Microsoft would like your feedback",
+  "Overall, how satisfied are you with the MSSQL extension?": "Overall, how satisfied are you with the MSSQL extension?",
```

Comment:
Was this phrasing used in vscode or ADS?  I think the "proper" wording is "would you recommend to a friend or colleague", and someone (idr who) expressed concern about a different prompt changing the resulting data.